ggml/src/ggml-cuda.cu

```diff
@@ -2345,6 +2349,8 @@ static bool ggml_cuda_compute_forward(ggml_backend_cuda_context & ctx, struct gg
         case GGML_OP_CROSS_ENTROPY_LOSS:
             ggml_cuda_cross_entropy_loss(ctx, dst);
             break;
+        case GGML_OP_RWKV_WKV:
+            ggml_cuda_op_rwkv_wkv(ctx, dst);
         case GGML_OP_CROSS_ENTROPY_LOSS_BACK:
```

Comment:
Is the fall through here intentional?

Reply:
Ah. This should be a merge error

Reply:
> Is the fall through here intentional?

Created #9589 to fix this. Thanks a lot for your help!